components/camel-netty-http/src/main/java/org/apache/camel/component/netty/http/handlers/HttpClientChannelHandler.java

```diff
@@ -75,6 +77,29 @@ protected Message getResponseMessage(Exchange exchange, ChannelHandlerContext ct
             producer.getEndpoint().getCookieHandler().storeCookies(exchange, uri, m);
         }
 
+        //handleNoContent(exchange, answer, response);
         return answer;
     }
+    
+//    protected void handleNoContent(Exchange exchange, Message answer, HttpResponse response) {
```

Comment:
I'll clean this up... just wanted to make sure had the right approach overall first